packages/engine/Source/Renderer/CubeMapFace.js

```diff
@@ -336,4 +333,84 @@ CubeMapFace.prototype.copyFromFramebuffer = function (
   gl.bindTexture(target, null);
   this._initialized = true;
 };
+
+/**
+ * Copies texels from the framebuffer to the cubemap's face mipmap.
+ * @param {number} [xOffset=0] An offset in the x direction in the framebuffer where copying begins from.
+ * @param {number} [yOffset=0] An offset in the y direction in the framebuffer where copying begins from.
+ * @param {number} [width=CubeMap's width] The width of the subimage to copy.
+ * @param {number} [height=CubeMap's height] The height of the subimage to copy.
+ * @param {number} [level=0] The level of detail. Level 0 is the base image level and level n is the n-th mipmap reduction level.
+ * @throws {DeveloperError} Cannot call copyFromFramebuffer when the texture pixel data type is FLOAT.
+ * @throws {DeveloperError} Cannot call copyFromFramebuffer when the texture pixel data type is HALF_FLOAT.
+ * @throws {DeveloperError} This CubeMap was destroyed, i.e., destroy() was called.
+ * @throws {DeveloperError} xOffset must be greater than or equal to zero.
+ * @throws {DeveloperError} yOffset must be greater than or equal to zero.
+ * @throws {DeveloperError} framebufferXOffset must be greater than or equal to zero.
+ * @throws {DeveloperError} framebufferYOffset must be greater than or equal to zero.
+ * @throws {DeveloperError} xOffset + source.width must be less than or equal to width.
+ * @throws {DeveloperError} yOffset + source.height must be less than or equal to height.
+ * @throws {DeveloperError} This CubeMap was destroyed, i.e., destroy() was called.
+ *
+ * @example
+ * // Copy the framebuffer contents to the +x cube map face.
+ * cubeMap.positiveX.copyFromFramebuffer();
+ */
+CubeMapFace.prototype.copyMipmapFromFramebuffer = function (
+  xOffset,
+  yOffset,
+  width,
+  height,
+  level,
+) {
+  xOffset = defaultValue(xOffset, 0);
```

Comment:
I'm going to leave this one here because `defaultValue` is already used extensively in this class, and I'm not sure which PR is going to be merged first. Will happily update if the other PR is merged first.